core/src/main/resources/schema/app/20180813131331_BUG-107734_cleanup-ldap-configs-for-org.sql

```diff
@@ -0,0 +1,12 @@
+-- // BUG-107734 cleanup-ldap-configs-for-org
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE ldapconfig ALTER COLUMN owner DROP NOT NULL;
+ALTER TABLE ldapconfig ALTER COLUMN account DROP NOT NULL;
+
+-- //@UNDO
+-- SQL to undo the change goes here.
+
+ALTER TABLE ldapconfig ALTER COLUMN owner SET NOT NULL;
```

Comment:
Some cases the migration will fail, because if the existing data is already null